test/system/organization_account_request_test.rb

```diff
@@ -0,0 +1,25 @@
+require "application_system_test_case"
+
+class NewOrganizationTest < ApplicationSystemTestCase
+  setup do
+    set_organization(nil)
+  end
+
+  context "request an account" do
+    should "direct to the new organization's form", js: true do
+      visit root_url
+      click_on "Request an account", match: :first
+
+      fill_in "Organization name", with: "New organization"
+      fill_in "Your name", with: "Doe Smith"
+      fill_in "Organization phone number", with: "12345678"
+      fill_in "Organization email", with: "organization@example.com"
+      select "France", from: "Country"
+      select "Corsica", from: "Province/State"
```

Comment:
I'm having a problem with the province selection in this test which is populated by the stimulus controller.

Reply:
I changed the province select field to a text field to avoid the system test error, as we are no longer building it through the location object.